docs/EnterpriseScaleAnalytics-KnownIssues.md

```diff
@@ -73,7 +73,7 @@ This error message appears, if the Purview soft-limit has been reached inside yo
 
 **Solution:**
 
-This error message appears, when deploying Purview and the `Microsoft.EventHub`, `Microsoft.Storage` and/or `Microsoft.Purview` Resource Provider has not been registered on the subscription or has been registered during an early Preview phase. We have released a fix to our Deploy To Azure Buttons to overcome this issue until it gets fixed on the Purview side. If you still run into this problem, please open an Issue in this repo and start (re-)registering the three Resource Providers manually through the `Azure Portal`, `Azure Powershell` or `Azure CLI` as mentioned [here](https://docs.microsoft.com/azure/azure-resource-manager/management/resource-providers-and-types). Once that is done, use the "Redeploy" button on the failed deployment to restart the deployment and successfuly deploy the components.
+This error message appears, when deploying Purview and the `Microsoft.EventHub`, `Microsoft.Storage` and/or `Microsoft.Purview` Resource Provider is not registered for the subscription or it was registered during early preview phase. If Purview resource provider is already registered, please un-register Purview resource provider and then re-register it. We have released a fix to our Deploy To Azure Buttons to overcome this issue until it gets fixed on the Purview side. If you still run into this problem, please open an Issue in this repo and start (re-)registering the three Resource Providers manually through the `Azure Portal`, `Azure Powershell` or `Azure CLI` as mentioned [here](https://docs.microsoft.com/azure/azure-resource-manager/management/resource-providers-and-types). Once that is done, use the "Redeploy" button on the failed deployment to restart the deployment and successfuly deploy the components.
```

Comment:
```suggestion
This error message appears, when deploying Purview and the `Microsoft.EventHub`, `Microsoft.Storage` and/or `Microsoft.Purview` Resource Provider (RP) is not registered for the subscription or it was registered during early preview phase. If the `Microsoft.Purview` RP is already registered, please un-register and re-register it. We have released a fix to our Deploy To Azure Buttons to overcome this issue until it gets fixed on the Purview side. If you still run into this problem, please open an Issue in this repo and start (re-)registering the three Resource Providers manually through the `Azure Portal`, `Azure Powershell` or `Azure CLI` as mentioned [here](https://docs.microsoft.com/azure/azure-resource-manager/management/resource-providers-and-types). Once that is done, use the "Redeploy" button on the failed deployment to restart the deployment and successfully deploy the components.
```

Reply:
@amanjeetsingh please see my suggestions above. I tried to make some of the words consistent like RP.

Reply:
Thanks @marvinbuss